includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -597,6 +596,8 @@ private function process_node( $node ) {
 					$is_bind_attribute = (
 						'[' === $name[0]
 						||
+						'data-amp-bind-' === substr( $name, 0, 40 )
```

Comment:
Should this use `AMP_BIND_DATA_ATTR_PREFIX` instead of hard-coding `data-amp-bind-`?